### DISCUSSION TITLE
2019/11/pdanet-dari-android-ke-ubuntu-tutorials

### DISCUSSION THREAD

**Discussion by giscus** [madi10/MANTANKODE#41] [General]
# 2019/11/pdanet-dari-android-ke-ubuntu-tutorials

Tutorial menggunakan internet suntik dari Android ke Laptop (Linux,Ubuntu dll)

https://www.mantankode.net/2019/11/pdanet-dari-android-ke-ubuntu-tutorials.html

**Comment by dnandz** [comment]
mantap kalo bisa versi terminal :)

**Reply by madi10** [reply]
belum nemu caranya saya, kalau ada boleh sharing disini.
